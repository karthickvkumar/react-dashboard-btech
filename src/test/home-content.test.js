import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
// import HomeContent from '../components/home-content';
import Sample from '../components/sample';
import renderer from 'react-test-renderer';

import ApiService  from '../services/api';

configure({adapter : new Adapter()});

test('Should return true', () => {
    //const component = shallow(<Sample />);
    const tree = renderer
    .create(<Sample />)
    .toJSON();
  expect(tree).toMatchSnapshot();
   // expect(component).toMatchSnapshot();
});

// describe('To check the validateEmail Method',() => {
//     test('Should return true', () => {
//         const component = shallow(<Sample />);
//         // expect(component.instance().validateEmail()).toBe(true);
//         expect(component).toMatchSnapshot();
//     });

//     test('Check the jest function', () => {
//         // console.log("Before Mocking", ApiService.validateEmail())
        
//         // ApiService.validateEmail = jest.fn(() => 5)
//         ApiService.validateEmail = jest.fn();
//         ApiService.validateEmail.mockReturnValue(10)

//         //console.log("After Mocking", ApiService.validateEmail());
//        expect(ApiService.validateEmail(10)).toBe(10);
//        expect(ApiService.validateEmail).toHaveBeenCalledWith(10);
//     })
// })