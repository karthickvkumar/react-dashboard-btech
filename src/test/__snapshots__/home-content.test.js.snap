// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should return true 1`] = `
<div>
  <table>
    <thead>
      <th>
        <tr>
          Name
        </tr>
        <tr>
          Age
        </tr>
      </th>
    </thead>
    <tbody />
  </table>
</div>
`;
